epoch_multinode_test.go

```diff
@@ -242,15 +248,52 @@ func (tw *testWAL) containsEmptyVote(round uint64) bool {
 	return false
 }
 
+// messageFilter defines a function that filters
+// certain messages from being sent or broadcasted.
+type messageFilter func(*Message, NodeID) bool
+
+// allowAllMessages allows every message to be sent
+func allowAllMessages(*Message, NodeID) bool {
+	return true
+}
+
+// denyFinalizationMessages blocks any messages that would cause nodes in
+// a network to index a block in storage.
+func denyFinalizationMessages(msg *Message, destination NodeID) bool {
+	if msg.Finalization != nil {
+		return false
+	}
+	if msg.FinalizationCertificate != nil {
+		return false
+	}
+	if msg.Notarization != nil {
```

Comment:
why do we also need to block notarizations? isn't it enough to block the above two?